PetScanList/wikitable.py

```
@@ -0,0 +1,59 @@
+def wiki_table(tab):
+    # ---
+    table_head = ["title"]
+    # ---
+    rows = []
+    # ---
+    for row, data in tab.items():
+        # print(data)
+        # {'touched': '20230327144419', 'Q': 'Q6853789', 'ns': 0, 'len': 3393, 'title': 'محمد حسين هيثم'}
+        # ---
+        if table_head == ["title"]:
+            table_head += [x for x in data.keys() if x != "title"]
```

Ruff (0.8.2)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)